aspnetcore/includes/razor.md

```diff
@@ -1,4 +1,10 @@
-This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a new alternative in ASP.NET Core 2.0 and later, a page-based programming model that makes building web UI easier and more productive. We recommend you try the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) tutorial before the MVC version. The Razor Pages tutorial:
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a feature of the ASP.NET Core MVC framework introduced in ASP.NET Core 2.0. Razor Pages programming:
+
+* Is page-based programming model that makes building web UI easier and more productive. 
+* Is similar to MVC.
+* Can be used with MVC controllers and views.
+
+After using both MVC and Razor Pages, many developers prefer Razor Pages. We recommend you try the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) tutorial before the MVC/Controller/Views version. The Razor Pages tutorial:
```

Comment:
"both MVC and Razor" no longer makes sense since Razor is a feature of MVC. 
Suggest something like:

After using Razor pages, many developers prefer it over controllers and views. ...